db/seeds.rb

```diff
@@ -1,7 +0,0 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
```

Comment:
I would still keep this file, which is what we use for seeding production (or even local database with production data) -- data that we want to keep

Reply:
I just eliminated the comments, the file is still there :)